t/db_dependent/api/v1/availability.t

```diff
@@ -187,23 +187,17 @@ subtest '/availability/biblio' => sub {
         $t->request_ok($tx)
           ->status_is(200)
           ->json_has('/0/availability')
-          ->json_is('/0/availability/available' => Mojo::JSON->false)
+          ->json_is('/0/availability/available' => Mojo::JSON->true)
```

Comment:
It was changed to false in the mentioned commit ae959a3, which actually broke several tests. Since anecdotal evidence points to the functionality actually working properly, my conclusion was that the test changes were borked.

Reply:
I also verified that the tests were working fine in the previous commit and broke with commit ae959a3.

Reply:
Ok. Signed off.